### ISSUE TITLE
Does not preserve paragraph breaks in highlighted text

### ISSUE THREAD

**Issue by Calorion** [getmatterapp/obsidian-matter#41]
**Original, in Matter:**

![image](https://user-images.githubusercontent.com/3050950/201496537-70df0c03-2053-4eef-a101-f670872f3266.png)

**In Obsidian: **

## Highlights
* When you invite a middle-aged moralist to address you, I suppose I must conclude, however unlikely the conclusion seems, that you have a taste for middle-aged moralising. I shall do my best to gratify it. I shall in fact, give you advice about the world in which you are going to live. I do not mean by this that I am going to talk on what are called current affairs. You probably know quite as much about them as I do. I am not going to tell you—except in a form so general that you will hardly recognise it—what part you ought to play in post-war reconstruction. It is not, in fact, very likely that any of you will be able, in the next ten years, to make any direct contribution to the peace or prosperity of Europe. You will be busy finding jobs, getting married, acquiring facts. I am going to do something more old-fashioned than you perhaps expected. I am going to give advice. I am going to issue warnings. Advice and warnings about things which are so perennial that no one calls them “current affairs.”

---

Please include paragraph breaks in the synced highlights. I understand that this presents formatting challenges, but I prefer that my highlights are rendered as blockquotes anyway, so a `>` before each paragraph works great for me.

**Comment by joeviezner** [comment]
Wouldn't that be handled by the highlights template? Could you insert a line break where required? Personally, I'm trying to get an empty line between the header and first highlight, in keeping with proper markdown linting. The thing is, I don't know what represents a linebreak in this templating language. I'm not a coder/developer. Here's my highlights template.

```
* {{text}} From Matter App
<---I want an empty line here--->
{% if note %}
  * **Note**: {{note}}
{% endif %}
```

**Comment by Calorion** [comment]
Yes, the highlights template fixes that, though I could recommend a change in the default template. I didn't know about that feature when I posted this.

**Comment by Calorion** [comment]
By the way, here's how you get a line break: `{{"\n"}}`

I'm not a coder/developer either, which is why I had to do a deep dive into the specs to figure this out!

Here are my templates, though I am not yet satisfied with them:

```
---
title: "{{title}}"
{% if author %}
author: "{{author}}"
{% elif publisher %}
author: "{{publisher}}"
{% endif %}
url: {{url}}
{% if published_date %}
published: {{published_date}}
{% endif %}
obsidianUIMode: preview
tags: 
 - {% for tag in tags %}{{tag | replace(' ', '_')}}{% if not loop.last %}{{"\n"}} - {% endif %}{% endfor%}

---

{% if author %}
[[{{author}}]]{% if publisher %}
, *{{publisher}}*  {% endif %}  
{% elif publisher %}
[[{{publisher}}]]  
{% endif %}
{% if published_date %}
{{published_date|date("MMM D, YYYY[ at ]h:mm [<span style='font-variant:small-caps;'>]a[</span>]") | replace('am', 'a.m.') | replace('pm', 'p.m.')}}
{% endif %}

<{{url}}>

***{% if note %}

## General Notes

{{note}}{% endif %}
```

```
{{"\n"}}### <span style='font-size: 0.75em;'>{{created_date|date("h:mm [<span style='font-variant:small-caps;'>]a[</span>,] MMMM D, YYYY") | replace('am', 'a.m.') | replace('pm', 'p.m.')}}</span>
>{{text}}
{% if note %}

{{note}}
{% endif %}
```

**Comment by joeviezner** [comment]
Fantastic! I love this highlights template and does exactly what I was looking for with the added bonus of introducing a nicely formatted sub-heading with date! Thanks for sharing!

Here's my metadata template I made which follows my Obsidian template for "Literature" notes. It makes use of the Templater plugin.

```
---
type: Literature
{% if tags %}
tags: [literature{% for tag in tags %} {{tag | replace(' ', '_')}}{% if not loop.last %}{% endif %}{% endfor%}]
{% endif %}
aliases:
  - {{title}}
url: "{{url}}"
{% if author %}
author: "{{author}}"
{% endif %}
{% if publisher %}
publisher: "{{publisher}}"
{% endif %}
{% if published_date %}
publishedDate: {{published_date}}
{% endif %}
{% if note %}
note: {{note}}
{% endif %}
created: <% tp.file.creation_date("dddd Do MMMM YYYY HH:mm:ss") %>
modified: <% tp.file.last_modified_date("dddd Do MMMM YYYY HH:mm:ss") %>
---
```

**Comment by Calorion** [comment]
Hm. I use Linter to insert created and modified dates, but Templater may be a better idea. Thanks.
